zjsonpatch/src/main/java/io/fabric8/zjsonpatch/internal/collections4/ListUtils.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.zjsonpatch.internal.collections4;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Ported from <a href=
+ * "https://github.com/apache/commons-collections/blob/master/src/main/java/org/apache/commons/collections4/ListUtils.java">Apache
+ * Commons Collections</a>
+ */
+public class ListUtils {
+  private ListUtils() {
+  }
+
+  public static <T> List<T> longestCommonSubsequence(final List<T> list1, final List<T> list2) {
```

Comment:
This means that all of the other commons-collections bulky classes are no longer needed? Nice

Reply:
Thanks to ChatGPT ;-)